src/app/reader/reader.component.ts

```diff
@@ -286,5 +292,10 @@ export class ReaderComponent implements OnInit, OnDestroy {
     this.scrollInformationService.initWatchParagraphs(element);
     this.ebookDisplayManagerService.updateContent(element, styleSheet);
     window.scrollTo(0, 0);
+    setTimeout(() => {
+      for (let index = 0, length = urls.length; index < length; index++) {
+        URL.revokeObjectURL(urls[index]);
+      }
+    });
```

Comment:
Reason for setTimeout?

Reply:
to kick it out from the current rendering micro task - otherwise you would revoke the url in the same tick as the browser would use to render the images (which means they can't be loaded then)